src/libs/swapAndBridge/swapAndBridge.ts

```diff
@@ -346,11 +346,23 @@ const addCustomTokensIfNeeded = ({
   return newTokens
 }
 
+// the celo native token is at an address 0x471EcE3750Da237f93B8E339c536989b8978a438
+// and LiFi doesn't work if we pass address 0 for this. We map it only for
+// lifi to make the swap work in this case
+const getLiFiTokenAddress = (chainId: number, tokenAddr: string) => {
+  if (tokenAddr !== ZeroAddress) return tokenAddr
+  // celo chain
+  if (chainId !== 42220) return tokenAddr
+
+  return '0x471EcE3750Da237f93B8E339c536989b8978a438'
+}
```

Comment:
Please make the comment public so it's displayed when the function is hovered:
![image](https://github.com/user-attachments/assets/5b754dad-858f-4f0f-a3c5-7119283d4a22)